modules/xmpp/xmpp.js

```diff
@@ -1110,5 +1110,16 @@ export default class XMPP extends Listenable {
         }
 
         this.sendDeploymentInfo = false;
+
+        const { region, shard } = aprops;
+
+        if (region || shard) {
+            // avoids sending empty values
+            this.eventEmitter.emit(JitsiConnectionEvents.PROPERTIES_UPDATED, JSON.parse(JSON.stringify({
```

Comment:
Suggestion: use lodash clone or structuredClone.

Reply:
That will drop the keys with undefined values?

Reply:
No. Do you want to?

Reply:
Yes. That was a quick hack to avoid if this and if that to not add the keys.

Reply:
Ah then go ahead!